package.nls.json

```diff
@@ -83,9 +86,9 @@
     "cmake-tools.configuration.cmake.generator.description": "The CMake generator to use.",
     "cmake-tools.configuration.cmake.toolset.description": "The CMake toolset to use when configuring.",
     "cmake-tools.configuration.cmake.platform.description": "The CMake platform to use when configuring.",
-    "cmake-tools.configuration.cmake.configureArgs.description": "Additional arguments to pass to CMake when configuring.",
-    "cmake-tools.configuration.cmake.buildArgs.description": "Additional arguments to pass to CMake when building.",
-    "cmake-tools.configuration.cmake.buildToolArgs.description": "Additional arguments to pass to the underlying build tool when building.",
+    "cmake-tools.configuration.cmake.configureArgs.description": "Additional arguments to pass to CMake when configuring. When using CMake Presets, these are to be used as temporary overrides.",
```

Comment:
As currently written, this could be interpreted as the value of `configureArgs` replacing any existing arguments ("override") when I think what actually happens is `configureArgs` is appended to the existing arguments. What if we reword to something like this?

> Additional arguments to pass to CMake when configuring. When using CMake Presets, these arguments are temporarily appended to the arguments provided by the active configure preset.

Reply:
I think that makes sense, I'll make similar changes for all of the affected strings.